src/app/app-routing.module.ts

```diff
@@ -35,6 +35,8 @@ import {HostAnalyticsComponent} from './components/analytics/host-analytics/host
 import {ResetPasswordComponent} from './components/auth/reset-password/reset-password.component';
 import {ResetPasswordConfirmComponent} from './components/auth/reset-password-confirm/reset-password-confirm.component';
 
+import { AuthGuard } from './guards/auth-guard.service';
```

Comment:
Can we have a better name:  Authentication/authorization?

Reply:
That is normal convention

Reply:
@Sanji515